docs/eventing/channels/create-default-channel.md

```diff
@@ -6,42 +6,61 @@ type: "docs"
 showlandingtoc: "true"
 ---
 
-# Creating a channel using cluster or namespace defaults
+Developers can create channels of any supported implementation type by creating an instance of a
+`Channel` object.
 
-Developers can create channels of any supported implementation type by creating an instance of a Channel object.
+To create a channel:
 
-1. Create a Channel object:
+1. Create a [Channel object](https://knative.dev/docs/reference/api/eventing/#messaging.knative.dev/v1.Channel): <!-- Where do you enter the below? -->
 
       ```yaml
       apiVersion: messaging.knative.dev/v1
       kind: Channel
       metadata:
-        name: my-channel
-        namespace: default
+        name: <my-channel>
+        namespace: <namespace>
       ```
+      Where:
+      * `<my-channel>` is a name of your choice.
+      * `<namespace>` is the name of your target namespace.
+
+      If you create this object in the `default` namespace, according to the default ConfigMap
+      example in [Channel types and defaults](./channel-types-defaults), it is an `InMemoryChannel`
+      channel implementation.
```

Comment:
Again, `InMemoryChannel` vs InMemoryChannel - be consistent and doc which formatting we use